exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -202,12 +218,14 @@ export default ExpFrameBaseComponent.extend({
       target.unshiftObject(card);
     },
     nextPage() {
+      this.set('cardSortResponse', Ember.copy(this.buckets, true));
```

Comment:
If `cardSortResponse` is meant to track the data in `buckets`, why not do that as a computed property instead of copying values?

Also, do you need to use `this.get('buckets')`? It looks like this may be setting a value to `undefined`.